tests/test_preprints.py

```diff
@@ -17,6 +17,9 @@
 
 logger = logging.getLogger(__name__)
 
+def find_current_browser(driver):
```

Comment:
@cslzchen, @felliott, I've used this method in `test_project_files.py`. Would this be a good opportunity to refactor?

Reply:
I agree. [This method](https://github.com/cos-qa/osf-selenium-tests/blob/d690e47a602b1834750599da80e598d400eabc30/tests/test_project_files.py#L80) doesn't belong to any specific test. We can move it out into some utility file to be imported by `test_preprints.py`, `test_project_files.py` and other potential tests in the future.

Reply:
I've moved it out to utils.py!